### ISSUE TITLE
Typescript Service died repeatedly

### ISSUE THREAD

**Issue by cisaacson** [microsoft/vscode#89306]
Issue Type: <b>Bug</b>

This issue was reported by VS Code, so I don't know how to reproduce it. I can see the "Initializing JS/TS language features" just keeps going. I am using Typescript 3.7.5

VS Code version: Code 1.41.1 (26076a4de974ead31f97692a0d32f90d735645c0, 2019-12-18T14:57:51.166Z)
OS version: Darwin x64 19.2.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-7500 CPU @ 3.40GHz (4 x 3400)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off<br>surface_control: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|4, 3, 3|
|Memory (System)|24.00GB (3.11GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (9)</summary>

Extension|Author (truncated)|Version
---|---|---
xml|Dot|2.5.0
type-script-preview|feb|0.0.1
flatbuffers|gab|0.1.0
vscode-edit-csv|jan|0.2.5
open-with-finder|maz|1.0.2
proto|pet|0.0.2
show-offset|ram|0.0.4
rust|rus|0.7.0
vscode-todo-highlight|way|1.0.4

</details>

**Comment by mjbvz** [comment]
Does this reproduce in the [latest VS Code insiders build](https://code.visualstudio.com/insiders/) with all [extensions disabled](https://github.com/microsoft/vscode/wiki/Performance-Issues#run-with-extensions-disabled)?

**Comment by cisaacson** [comment]
I am not using the latest insiders build. However, I did restart VS Code and it is not happening any longer.

The only other thing I can think of that may contributed is I was running a test in debug mode (a mocha test with a very long timeout) and walked away from my desk. Perhaps that caused it.

The only reason I reported this is that VS Code suggested I do so, hopefully it's of some use to track it. I am not blocked by this.
